### DISCUSSION TITLE
new member request

### DISCUSSION THREAD

**Discussion by mitegab** [EddieHubCommunity/support#8722] [General]
Guys please I want to be a member of this community. please add me

**Comment by angelskydiver** [comment]
hi. give me a chance to collaborate.

**Comment by dinxsh** [comment]
hey @proangel1215 @mitegab , you can join eddiehub org by raising an issue [here](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) and filling up the necessary details, you'll be provided the join link shortly through an automated process

**Reply by angelskydiver** [reply]
thank you.

**Comment by griffin-k** [comment]
I want to be a member of this community kindly add me

**Reply by dinxsh** [reply]
make an issue [here](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization)
